src/fixtures/create_test_accounts.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO
+  account (accountId, primaryEmail, status, notificationPreferences, type)
+VALUES
+  (2, 'test@permanent.org', 'status.auth.ok', '{}', 'type.account.standard'),  
+  (3, 'test+1@permanent.org', 'status.auth.ok', '{}', 'type.account.standard');  
```

Comment:
Any reason not to make these 'type.account.test'?

Reply:
I think that as long as we're not actually going to need to filter test accounts out of analysis in a real environment, it's ideal for them to look as much like real accounts as possible.